test/integration/epp/domain/info/base_test.rb

```diff
@@ -25,7 +25,11 @@ def test_returns_valid_response
          headers: { 'HTTP_COOKIE' => 'session=api_bestnames' }
 
     response_xml = Nokogiri::XML(response.body)
+
+    schema_version = return_xml_domain_schema_version(response_xml)
+   
     assert_epp_response :completed_successfully
+    assert schema_version >= 1.1
```

Comment:
I'd suggest to create dedicated assertion method using return_xml_domain_schema_version(response_xml) as a private method call, so instead of two lines 
```
schema_version = return_xml_domain_schema_version(response_xml)
assert schema_version >= 1.1
```
we could just do something like
```
assert schema_version_bigger_for('domain', 1.1)
```